examples/Cpp/PrimitiveRestart/Example.cpp

```diff
@@ -0,0 +1,177 @@
+/*
+ * Example.cpp (Example_PrimitiveRestart)
+ *
+ * This file is part of the "LLGL" project (Copyright (c) 2015-2019 by Lukas Hermanns)
+ * See "LICENSE.txt" for license information.
+ */
+
+#include <ExampleBase.h>
+#include <vector>
+
+class Example_PrimitiveRestart : public ExampleBase
```

Comment:
Feel free to add your name to this example in a comment like `// author: ...` if you want to :)